portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,28 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  ArrayList<String> comments = new ArrayList<String>();
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Iris!</h1>");
+    comments.add("Hi");
```

Comment:
Not sure if this is intentional, but this means that every time a request is made to your server, more comments are added to your list. In other words, the endpoint is not a simple "fetch me the data on server", it's a "add more data, then fetch the updated data".

If you only want to add these once, I'd move these comments.add statements out of the method.